WcaOnRails/spec/models/registration_spec.rb

```diff
@@ -54,13 +54,13 @@
   end
 
   it "requires at least one event" do
-    registration.eventIds = ""
+    registration.registration_events.destroy_all
```

Comment:
Actually, it seems that the last think is something normal for Rails' relations, I wasn't aware of it =)